### DISCUSSION TITLE
ERROR: Wheel 'torch' located at C:\Users\(User)\AppData\Local\Temp\pip-unpack-wuc4qyzp\torch-1.13.0+cu116-cp310-cp310-win_amd64.whl is invalid.

### DISCUSSION THREAD

**Discussion by waywardgalaxy** [openai/whisper#546] [Q&A]
Following this thread here ([https://github.com/openai/whisper/discussions/47]), trying to get my GTX 970 to run whisper rather than CPU. 

However received this shortly into the download:
ERROR: Wheel 'torch' located at C:\Users\(User)\AppData\Local\Temp\pip-unpack-wuc4qyzp\torch-1.13.0+cu116-cp310-cp310-win_amd64.whl is invalid.

Any thoughts? I'm a novice programmer.

**Comment by zero-stroke** [comment]
Hi, did you ever figure this out?
